packages/ember-views/lib/system/build-component-template.js

```diff
@@ -135,17 +160,32 @@ function normalizeComponentAttributes(component, isAngleBracket, attrs) {
       if (colonIndex !== -1) {
         var attrProperty = attr.substring(0, colonIndex);
         attrName = attr.substring(colonIndex + 1);
-        expression = ['get', 'view.' + attrProperty];
+
+        if (attrName === 'type' && hardCodeType) {
+          expression = component.get(attrProperty);
```

Comment:
@mixonic and I discussed this in chat.  When set to a string HTMLBars `manualElement` calls `dom.setAttribute` **before** appending to the fragment, any value other than a string here would result in the element being created and appended to a fragment, *then* the attributes would be set which would trigger the error on IE8.

Relevant code paths:

* https://github.com/tildeio/htmlbars/blob/master/packages/htmlbars-runtime/lib/render.js#L86-L87
* https://github.com/tildeio/htmlbars/blob/master/packages/htmlbars-runtime/lib/render.js#L108-L109